native/src/services/__tests__/createSettingsSections.spec.ts

```diff
@@ -81,17 +81,19 @@ describe('createSettingsSections', () => {
       if (pushNotificationSection) {
         pushNotificationSection.onPress()
       }
-      const settings = defaultSettings
-      settings.allowPushNotifications = false
-      const changedSettings = changeSetting(settings)
-      // @ts-ignore
-      expect(pushNotificationSection.getSettingValue(settings)).toBeFalsy()
-      expect(changedSettings.allowPushNotifications).toBeTruthy()
-      settings.allowPushNotifications = true
-      const changedSettings2 = changeSetting(settings)
-      // @ts-ignore
-      expect(pushNotificationSection.getSettingValue(settings)).toBeTruthy()
-      expect(changedSettings2.allowPushNotifications).toBeFalsy()
+      if (!pushNotificationSection?.getSettingValue) {
+        expect(true).toBeFalsy()
+      } else {
+        const settings = defaultSettings
+        settings.allowPushNotifications = false
+        const changedSettings = changeSetting(settings)
+        expect(pushNotificationSection.getSettingValue(settings)).toBeFalsy()
```

Comment:
If I interpret this right:
```
if (!pushNotificationSection?.getSettingValue) {
  expect(true).toBeFalsy()
}
```
you want to tell typescript that pushNotificationSection is set (probably otherwise typescript says it can be undefined)
You could simply tell typescript that it is set:
```
pushNotificationSection!.getSettingValue(settings)
```
This unit test will fail with "Cannot read property 'getSettingValue' of undefined" if this is not the case